compiler/nim.cfg

```diff
@@ -16,7 +16,7 @@ define:nimcore
 define:useStdoutAsStdmsg
 
 @if nimHasStyleChecks:
-  styleCheck:error
+  styleCheck:off
```

Comment:
add a PRTEMP or FIXME or whatever your favorite single token to avoid forgetting to undo this (and other temporary) changes before PR can be merged

Reply:
Temporary due to the `N` versus `n` stylecheck debacle mentioned in IRC yesterday.

Reply:
i know, temporary debugging is totally fine, im just suggesting to add a unique token to remember (and let reviewers know) to remove this before PR can be merged, eg:
```nim
@if nimHasStyleChecks:
  styleCheck:off # PRTEMP
@endif
```
=> makes it easy to search all such places w grep and it tells reviewer to disregard that change